lib/datadog/appsec/context.rb

```diff
@@ -55,6 +55,22 @@ def run_rasp(type, persistent_data, ephemeral_data, timeout = WAF::LibDDWAF::DDW
         result
       end
 
+      def mark_as_interrupted!
+        @interrupted = true
+      end
+
+      def interrupted?
+        @interrupted == true
+      end
+
+      def ruleset_version
```

Comment:
If I will read that sentence `"Context ruleset version"` I would ask myself - what kind of ruleset version that context is talking about? I think we should prefix encapsulated methods we expose to make it clear what we are talking about.

Same goes for waf addresses, I do think we might give it a better name, like `waf runner supported addresses` or so. yes it's long, but it's clear